02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -29,9 +29,9 @@ Do not misinterpret the below table as a linear progression from Curious Outside
 | Potential User | Getting Started | Teaching a child how to cook | <ul><li>Focuses on the learner 'doing' stuff, not 'explaining' stuff to the learner</li><li>Provides a small simple working example that teaches the basics</li><li>New learners experience an 'I can use this now!' moment by the end</li><li>Focuses on concrete tasks, not abstract concepts</li><li>Does not use jargon</li><li>Explains only what is necessary and cuts out all else</li><li>Avoids explaining deeper concepts or different ways of doing the same thing</li></ul>
 | New User | How-To Guides | Following a cookbook's recipe | <ul><li>Achieves some goal or solves a problem</li><li>States the pre-requisites one needs to have before starting (not a Getting Started Guide)</li><li>The Guide follows a clearly-labeled step-by-step process</li><li>By following the steps, one reproduces the same results without fail</li><li>Explains the different ways one can achieve the same goal</li><li>Explains only what is necessary</li></ul>
 | Active User | Reference | Reading an encyclopedia | <ul><li>Concise explanation of each piece of the code</li><li>The structure of the reference mirrors the structure of the code it documents</li><li>Formatting is consistent throughout the material</li></ul>
-| Experienced User | Explanation | Listening to a CEO answer questions about his company | <ul><li>Explains the context/history</li><li>Explains the significant design decisions made, their alternativees, and the reasons one was chosen over another</li><li>Implies where things could be improved, expanded, refined, etc.</li></ul>
+| Experienced User | Explanation | Listening to a CEO answer questions about his company | <ul><li>Explains the context/history</li><li>Explains the significant design decisions made, their alternatives, and the reasons one was chosen over another</li><li>Implies where things could be improved, expanded, refined, etc.</li></ul>
 
-Moreover, there are clear examples of "bad" documentation (as explained in [Teach, Don't Tell](http://stevelosh.com/blog/2013/09/teach-dont-tell/)):
+It's also important to take note of forms of documentation which are clearly "bad", (as explained in [Teach, Don't Tell](http://stevelosh.com/blog/2013/09/teach-dont-tell/)):
```

Comment:
Could you explain why you think this rendering is better? 
My main issue with your rendering is that it is longer than mine and perhaps less skimmable.

Reply:
My concern is saying that something is "clearly bad", which is a value judgement. Is our "bad" the same as the author's "bad"? I think this depends on goals, values, and principles. Have we clearly stated what ours are? At this stage, I think we have not.

I wanted soften the declaration of "bad", because it's a bit of a reflex for me, but thinking again now I see my issue with this declaration is a bit deeper. I'll spend more time looking into this and suggest a different change. I might just roll back this change - I'll comment here when I've decided.

Reply:
I made a revision which alleviates my concern with this part. Let me know what you think.

Reply:
I like that rendering a lot better.